### PR TITLE
Multi GPU mining - in conductor

### PR DESCRIPTION
**Pull request by hanssv**
Support multiple miner processes in the conductor rather than hacking it in the miner.

### CODE REVIEW

File: apps/aecore/src/aec_conductor.erl [hunk 1]
```diff
@@ -820,22 +871,32 @@ handle_micro_sleep_reply(ok, State) ->
...
+    start_mining(State);
 create_key_block_candidate(#state{top_block_hash = TopHash,
                                   beneficiary    = Beneficiary} = State) ->
     epoch_mining:info("Creating key block candidate on the top"),
     Fun = fun() ->
-              {aec_block_key_candidate:create(TopHash, Beneficiary), TopHash}
+                  {aec_block_key_candidate:create(TopHash, Beneficiary), TopHash}
           end,
-    dispatch_worker(create_key_block_candidate, Fun, State).
+    {State1, _Pid} = dispatch_worker(create_key_block_candidate, Fun, State),
```
Comment (sennui) [inline apps/aecore/src/aec_conductor.erl]: I see you are matching now on `{State, Pid}`, but top branch of case in dispatch_worker returns State?!

Comment (hanssv) [reply apps/aecore/src/aec_conductor.erl]: Well, we should never hit that one, but fixing it anyway... :-P

**Comment by hanssv** [comment]
Just pressed the ones GitHub suggested 😂 It was late in the afternoon...

File: docs/cuda-miner.md [hunk 2]
```diff
@@ -78,7 +78,19 @@ mining:
             hex_encoded_header: true
 ```
 
-After configuration could be started (or restarted if it's already running):
+If you have multiple GPU cards they can be addressed using the `-d <n>` flag with the cuda-miner. So if you want to use multiple cards
```
Comment (lucafavatella) [inline docs/cuda-miner.md]: Shall this addition rather be made at end of section "Multiple GPU devices"?

**Pull request merged**